### ISSUE TITLE
Failed on searching for invalid Jira issue key

### ISSUE THREAD

**Issue by ebronson68** [btwrk/action-jira-linter#195]
When running the Jira linter on a branch named, `2023-04-11-245a825`, the Jira linter incorrectly finds `11-245` to be the Jira issue key. This causes the action to fail as it cannot find a ticket matching the issue key in Jira.

We are aware the branch doesn't include a correct Jira issue key. However, this is a false positive as normally, the Jira linter would fail only if `fail-on-error` is set to `true`

Action logs:
```
Run jira-tools/action-jira-linter@v1.0.1
  with:
    github-token: ***
    jira-token: ***
    jira-base-url: https://***/
    pr-threshold: 1000
    jira-user: ***
    validate-issue-status: true
    allowed-issue-statuses: New
  Reopened
  To Do
  In Progress
  Peer Review
  In Staging / QA
  Ready For Staging
  In Development Env
  Approved
  Resolved
  
    skip-branches: ^(main|development|staging|production|dev)$
    fail-on-error: false
    skip-comments: true
Base branch ->  development
Head branch ->  2023-04-11-245a825
branch '2023-04-11-245a825' is not a default branch and does not match ignore pattern provided in 'skip-branches' option: /^(main|development|staging|production|dev)$/
JIRA key -> 11-245
{
  error: [AxiosError: Request failed with status code 404] {
    code: 'ERR_BAD_REQUEST',
    config: {
      transitional: [Object],
      adapter: [Function: httpAdapter],
      transformRequest: [Array],
      transformResponse: [Array],
      timeout: 2000,
      xsrfCookieName: 'XSRF-TOKEN',
      xsrfHeaderName: 'X-XSRF-TOKEN',
      maxContentLength: -1,
      maxBodyLength: -1,
      env: [Object],
      validateStatus: [Function: validateStatus],
      headers: [Object],
      baseURL: 'https://***/rest/api/3',
      method: 'get',
      url: '/issue/11-245?fields=project,summary,issuetype,labels,status,customfield_10016',
      data: undefined
    },
    request: ClientRequest {
      _events: [Object: null prototype],
      _eventsCount: 7,
      _maxListeners: undefined,
      outputData: [],
      outputSize: 0,
      writable: true,
      _last: true,
      chunkedEncoding: false,
      shouldKeepAlive: false,
      _defaultKeepAlive: true,
      useChunkedEncodingByDefault: false,
      sendDate: false,
      _removedConnection: false,
      _removedContLen: false,
      _removedTE: false,
      _contentLength: 0,
      _hasBody: true,
      _trailer: '',
      finished: true,
      _headerSent: true,
      socket: [TLSSocket],
      connection: [TLSSocket],
      _header: 'GET /rest/api/3/issue/11-245?fields=project,summary,issuetype,labels,status,customfield_10016 HTTP/1.1\r\n' +
        'Accept: application/json, text/plain, */*\r\n' +
        'authorization: Basic YW11X2RlcGxveUBhbXVuaXZlcnNhbC5jb206***\r\n' +
        'User-Agent: axios/0.27.2\r\n' +
        'Host: ***\r\n' +
        'Connection: close\r\n' +
        '\r\n',
      _keepAliveTimeout: 0,
      _onPendingData: [Function: noopPendingOutput],
      agent: [Agent],
      socketPath: undefined,
      method: 'GET',
      insecureHTTPParser: undefined,
      path: '/rest/api/3/issue/11-245?fields=project,summary,issuetype,labels,status,customfield_10016',
      _ended: true,
      res: [IncomingMessage],
      aborted: false,
      timeoutCb: null,
      upgradeOrConnect: false,
      parser: null,
      maxHeadersCount: null,
      reusedSocket: false,
      host: '***',
      protocol: 'https:',
      _redirectable: [Writable],
      [Symbol(kCapture)]: false,
      [Symbol(kNeedDrain)]: false,
      [Symbol(corked)]: 0,
      [Symbol(kOutHeaders)]: [Object: null prototype]
    },
    response: {
      status: 404,
      statusText: 'Not Found',
      headers: [Object],
      config: [Object],
      request: [ClientRequest],
      data: [Object]
    }
  }
}
Error: Request failed with status code 404
```

**Comment by andrewlarioza** [comment]
I been getting the same error, Is this still active? I'm not seeing any recent updates for the project.
